controllers/virtualmachinewebconsolerequest/v1alpha1/webconsolerequest_intg_test.go

```diff
@@ -97,19 +98,17 @@ func intgTestsReconcile() {
 			},
 		}
 
-		intgFakeVMProvider.Lock()
 		intgFakeVMProvider.GetVirtualMachineWebMKSTicketFn = func(ctx context.Context, vm *vmopv1.VirtualMachine, pubKey string) (string, error) {
```

Comment:
I think the fake FooFn assignments are protected by the lock because in the past the race detector has tripped with the Reconciler reading the field.